gnmi_server/server_test.go

```diff
@@ -3174,17 +3198,23 @@ func TestClient(t *testing.T) {
     }
 
     sdc.C_init_subs(true)
-
+    var gotNotiMu sync.Mutex
     for testNum, tt := range tests {
+        mu_hb_done.Lock()
         heartbeat = 0
+        mu_hb_done.Unlock()
+		mu_idx_done.Lock()
         event_index = 0
-        deinit_done = false
+        mu_idx_done.Unlock()
+		deinit_done = false
```

Comment:
Indentation is off.

Reply:
Thanks for your comments, I've fixed all the indent and naming issue.